Formula/pcp.rb

```diff
@@ -0,0 +1,18 @@
+class Pcp < Formula
+  desc "📦 Command-line peer-to-peer data transfer tool based on libp2p"
+  homepage "https://github.com/dennis-tra/pcp"
+  url "https://github.com/dennis-tra/pcp/archive/refs/tags/v0.3.3.tar.gz"
+  sha256 "058d4cc3d85b4af8875e2e2f043d0985aaf20acb193bb84275beccf8070e0bcd"
+  license "Apache-2.0"
+  head "https://github.com/dennis-tra/pcp.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/pcp/pcp.go"
+  end
+
+  test do
+    exec bin/"pcp", "help"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yup, you're right, I'll provide a better test 👍

Reply:
@carlocab Hmm, I honestly can't come up with a reasonable test case :/ would it suffice to match the help output for the version or similar?

Reply:
Can we attempt a transfer we know won't work and check for an expected error message and exit code?

Reply:
Right, I'll give that a try 👍

Reply:
@carlocab Now the test tries to transfer a file with words that definitely don't exist and thus it fails.

Reply:
It actually fails because you have lines that are too long. You also need to pass the expected exit code as the second argument of `shell_output`. It expects success by default and will fail if it doesn't get that.